### ISSUE TITLE
Is there a big difference in efficiency between parsing an article as a whole or as a paragraph by paragraph?

### ISSUE THREAD

**Issue by swanfly** [markedjs/marked#3548]
Is there a big difference in efficiency between parsing an article as a whole or as a paragraph by paragraph?

**Comment by UziTech** [comment]
Nope. Since we stop at certain things like blank lines for paragraphs, parsing one article that has 10 paragraphs will essentially parse each paragraph in series.

This behavior can change with certain extensions so we don't guarantee any speed when using extensions.

**Comment by swanfly** [comment]
okay, thank you
